mobile/src/main/java/org/jraf/android/cinetoday/mobile/api/codec/movie/MovieCodec.java

```diff
@@ -68,7 +69,7 @@ public void fill(Movie movie, JSONObject jsonMovie) throws ParseException {
                 try {
                     movie.releaseDate = Api.SIMPLE_DATE_FORMAT.parse(releaseDateStr);
                 } catch (java.text.ParseException e) {
-                    throw new ParseException(e);
+                    Log.d(e, "Invalid releaseDate %s in movie %s", movie, releaseDateStr);
```

Comment:
This fixes the initial bug: now the movie will not have a release date, but this will not block the parsing of the rest of the movies.

But for the specific movie we saw, in fact the release date is valid inside the movie list.  Since we don't overwrite it here, this movie will actually have a release date.